### DISCUSSION TITLE
Memory

### DISCUSSION THREAD

**Discussion by karthikeya21** [perspective-dev/perspective#2100] [Q&A] [answered]
## Support Question

..is there a way to reduce memory space while cloning github repositories into the system?

**Comment by sinistersnare** [comment, marked as answer]
when you clone you can do `git clone -–depth 1` to not download the whole history. You can also try [`gitoxide`](https://github.com/Byron/gitoxide/).

However, this question is better suited to a more general `git` forum, and not this.
